test/consul_enterprise_test.go

```diff
@@ -0,0 +1,17 @@
+package test
+
+import (
+	"os"
+	"testing"
+)
+
+// Test the example in the root folder
+func TestConsulInstallFromURLWithUbuntuAmi(t *testing.T) {
+	t.Parallel()
+	runConsulClusterTest(t, "ubuntu16-ami", ".", "../examples/consul-ami/consul.json", "ubuntu", os.Getenv("CONSUL_AMI_TEMPLATE_VAR_DOWNLOAD_URL"))
```

Comment:
See my comment in the Vault PR about adding a separate method that reports a clear error if this env var is not set and has a comment explaining why we use an env var.